src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.html

```diff
@@ -624,7 +633,16 @@ <h2 class="color-blue" *ngIf="showQuestion.length === 0 || showQuestion.includes
             which are <strong>Response Count [Weight of the choice]</strong>. Based on the weight assigned to the choice and
             the response count of that choice, total and average points for the sub-question is calculated for each recipient.
           </p>
-          <p>EXAMPLE BOX</p>
+          <tm-example-box>
+              <tm-instructor-session-result-question-view
+                  [questions]="exampleRubricQuestionQuestions"
+                  [section]="" [sectionType]="InstructorSessionResultSectionType.EITHER"
+                  [session]="exampleFeedbackSession"
+                  [showStatistics]="true" [indicateMissingResponses]="true"
+                  [isDisplayOnly]="true"
+                  [(instructorCommentTableModel)]="exampleInstructorCommentTableModel">
```

Comment:
Does the model need to be with the curvy brackets or would square brackets suffice?
```suggestion
                  [instructorCommentTableModel]="exampleInstructorCommentTableModel">
```